examples/nfct_test.rs

```diff
@@ -0,0 +1,170 @@
+#![no_std]
+
+extern crate alloc;
+extern crate lang_items;
+extern crate libtock_drivers;
+
+use core::fmt::Write;
+use libtock_core::result::CommandError;
+use libtock_drivers::console::Console;
+use libtock_drivers::nfc::NfcTag;
+use libtock_drivers::nfc::RecvOp;
+use libtock_drivers::result::FlexUnwrap;
+use libtock_drivers::result::TockError;
+use libtock_drivers::timer;
+use libtock_drivers::timer::Timer;
+use libtock_drivers::timer::Timestamp;
+
+#[allow(dead_code)]
+/// Helper function to write on console the received packet.
+fn print_rx_buffer(buf: &mut [u8], amount: usize) {
+    if amount < 1 || amount > buf.len() {
```

Comment:
Since `amont` is `usize`, it's probably more readable if you check `== 0` instead of `< 1`.